daily-challenges/2024-11/29/min241129.spec.ts

```
@@ -0,0 +1,40 @@
+
+test('Decoration', async ({ page }) => {
+    await page.goto('https://material.playwrightvn.com/')
+    await page.locator(`//a[normalize-space(text())='Game 04: Trang trí cây thông Noel']`).click()
+    const mission = []
+    const rows = await page.locator(`//div[@class='mission']//ul/li`).count()
+    for (let i = 1; i <= rows; i++) {
+        const items = await page.locator(`//div[@class='mission']//ul/li[${i}]`).innerText()
+        const parts = items.split(' ')
+        let qty = parseInt(parts[1])
+        let name = parts.slice(2).join(' ')
+        name = name.split(' ').map(word => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()).join(' ')
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'qty' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)